.github/workflows/cli.yml

```diff
@@ -0,0 +1,72 @@
+name: CLI
+
+on:
+  push:
+    branches: [develop]
+  pull_request:
+    branches: [develop]
+
+jobs:
+  build:
+    name: Build ${{ matrix.os }}
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        os: [linux]
+    steps:
+      - name: Install dependencies
+        run: sudo apt-get update && sudo apt-get install git -y
+      - name: Set up Go 1.13
```

Comment:
Should we pull the go version from https://github.com/mozilla/sops/blob/master/go.mod#L3 ?

Reply:
It's not trivial unless we generate the workflow yaml from that other file. I think for now it's fine to leave as is. That's how it was for the travisci config.

Reply:
Ok sounds good to me.